src/main/java/org/sonarqube/gradle/SonarQubeTask.java

```diff
@@ -38,32 +43,61 @@
  */
 public class SonarQubeTask extends DefaultTask {
 
-  private Map<String, Object> sonarProperties;
+    private static final Logger LOGGER = Logging.getLogger(SonarQubeTask.class);
+
+    public static final LogOutput LOG_OUTPUT = new LogOutput() {
+        @Override
+        public void log(String formattedMessage, Level level) {
+            switch (level) {
+                case TRACE:
+                    LOGGER.trace(formattedMessage);
+                    return;
+                case DEBUG:
+                    LOGGER.debug(formattedMessage);
+                    return;
+                case INFO:
+                    LOGGER.info(formattedMessage);
+                    return;
+                case WARN:
+                    LOGGER.warn(formattedMessage);
+                    return;
+                case ERROR:
+                    LOGGER.error(formattedMessage);
+                    return;
+                default:
+                    throw new IllegalArgumentException(level.name());
+            }
+        }
+    };
 
-  @TaskAction
-  public void run() {
-    Map<String, Object> properties = getProperties();
+    private Map<String, Object> sonarProperties;
 
-    Properties propertiesObject = new Properties();
-    propertiesObject.putAll(properties);
+    @TaskAction
+    public void run() {
+        Map<String, Object> properties = getProperties();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AIndentationCheck)